docs/source/developer/freeradius.rst

```diff
@@ -2,7 +2,7 @@
 Installation and configuration of Freeradius 3
 ==============================================
 
-This guide explains how to install and configure `freeradius 3 <http://freeradius.org/version3.html>`_
+This guide explains how to install and configure `freeradius 3 <https://freeradius.org>`_
```

Comment:
`http://freeradius.org/version3.html` no more exists, it redirects to `https://freeradius.org/releases/`